src/app/view/components/_common/iframe/iframe-porlet-legacy/services/iframe-events-handler.service.ts

```diff
@@ -21,15 +22,17 @@ export class DotIframeEventsHandler {
         private dotRouterService: DotRouterService,
         private dotContentletEditorService: DotContentletEditorService,
         private dotUiColorsService: DotUiColorsService,
-        private dotIframeService: DotIframeService
+        private dotIframeService: DotIframeService,
+        private dotPushPublishDialogService: DotPushPublishDialogService
     ) {
         if (!this.handlers) {
             this.handlers = {
                 'edit-page': this.goToEditPage.bind(this),
                 'edit-contentlet': this.editContentlet.bind(this),
                 'edit-task': this.editTask.bind(this),
                 'create-contentlet': this.createContentlet.bind(this),
-                'company-info-updated': this.setDotcmsUiColors.bind(this)
+                'company-info-updated': this.setDotcmsUiColors.bind(this),
+                'push-publish-dialog': this.pushPublishDialog.bind(this)
```

Comment:
```suggestion
                'push-publish': this.pushPublishDialog.bind(this)
```

Is the name of the event not the component.